### DISCUSSION TITLE
What TypeScript type is tw`...` ?

### DISCUSSION THREAD

**Discussion by k4y4k** [ben-rogerson/twin.macro#316] [Q&A] [answered]
Howdy :wave:

I have this code: 
```
const subtitleStyles = () => [tw`self-start p-2 bg-black`]  
```

and ESLint is yelling at me about annotating a return type. Sure. Fine. OK. But, uh, what is `tw` *meant* to be?

Using Gatsby and styled-components, everything's set up OK and it all works, I'm just removing linter errors :smile:

**Comment by k4y4k** [comment, marked as answer]
My Intellisense revealed that `TwStyle` is a type, so I guess I'm using that :thinking: Would welcome anyone else's input on this, though.

**Reply by ben-rogerson** [reply]
You found the answer, and yeah I should probably document that type.

**Reply by k4y4k** [reply]
"Yeah I should probably document `___`" is such a relatable admission I can't help but smile.
